src/QuiqChatClient.js

```diff
@@ -140,13 +135,19 @@ class QuiqChatClient {
     return this.textMessages;
   };
 
-  /*** API wrappers: these return Promises around the API response ***/
+  /** API wrappers: these return Promises around the API response **/
 
   joinChat = () => {
+    set(quiqChatVisible.id, 'true', {
+      expires: quiqChatVisible.expiration,
+    });
     return API.joinChat();
   };
 
   leaveChat = () => {
+    set(quiqChatVisible.id, 'false', {
```

Comment:
Does this mean that if a customer starts a chat and then closes it and then all the agents go to AFE, the chat won't show up if they refresh?